packages/web-app-ocm/src/views/IncomingInvitations.vue

```diff
@@ -0,0 +1,208 @@
+          <oc-text-input
+            ref="tokenInput"
+            v-model="token"
+            :label="$gettext('Enter invite token')"
+            :clear-button-enabled="true"
+          />
+          <br />
+          <oc-select
+            v-model="provider"
+            :label="$gettext('Select institution of inviter')"
+            :options="providers"
+            class="oc-mb-m"
+            :loading="loading"
+          >
+            <template #option="{ full_name, domain }">
+              <span class="option">
+                <strong v-text="full_name" />
+              </span>
+              <span class="option" v-text="domain" />
+            </template>
+            <template #no-options> No institutions found with this name </template>
+            <template #selected-option="{ full_name, domain }">
+              <strong class="oc-mr-s" v-text="full_name" />
+              <small v-text="domain.length > 17 ? domain.slice(0, 20) + '...' : domain" />
+            </template>
+          </oc-select>
+          <div v-if="providerError" class="oc-text-input-message">
+            <span class="oc-text-input-danger">{{ providerErrorMessage }}</span>
+          </div>
+        </div>
+        <br />
+        <br />
+        <oc-button size="small" :disabled="acceptInvitationButtonDisabled" @click="acceptInvite">
+          <oc-icon name="add" />
+          <span v-text="$gettext('Accept invitation')" />
+        </oc-button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, onMounted, ref, unref, VNodeRef } from 'vue'
+import {
+  useStore,
+  queryItemAsString,
+  useClientService,
+  useRoute,
+  useRouter,
+  useConfigurationManager
+} from '@ownclouders/web-pkg'
+import { $gettext } from '@ownclouders/web-pkg/src/router/utils'
+import { useGettext } from 'vue3-gettext'
+import { onBeforeRouteUpdate, RouteLocationNormalized } from 'vue-router'
+
+export default defineComponent({
+  emits: ['highlightNewConnections'],
+  setup(props, { emit }) {
+    const store = useStore()
+    const router = useRouter()
+    const clientService = useClientService()
+    const configurationManager = useConfigurationManager()
+    const { $gettext } = useGettext()
+
+    const token = ref(undefined)
+    const provider = ref(undefined)
+    const providers = ref([])
+    const loading = ref(true)
+    const providerError = ref(false)
+    const tokenInput = ref<VNodeRef>()
+
+    const helperContent = computed(() => {
+      return {
+        text: $gettext('Once you accept invitation, the inviter will be added to your connections.')
+      }
+    })
+
+    const acceptInvitationButtonDisabled = computed(() => {
+      return !unref(token) || !unref(provider) || unref(provider).full_name === 'Unknown provider'
+    })
+
+    const providerErrorMessage = computed(() => {
+      return isMyProviderSelectedProvider(unref(provider))
+        ? $gettext('You cannot select your institution')
```

Comment:
```suggestion
        ? $gettext('You cannot select your own institution')
```

Reply:
Thinking about it ... why would it be in the list then at all?!
Let's remove it from the list instead...